src/command_modules/azure-cli-hdinsight/azure/cli/command_modules/hdinsight/_help.py

```diff
@@ -47,16 +47,16 @@
           text: |-
               az hdinsight application create -n MyCluster -g MyResourceGroup \\
               --application-name MyApplication \\
-              --script-uri https://path/to/install/script.sh
+              --script-uri "https://path/to/install/script.sh" \\
               --script-action-name MyScriptAction \\
-              --script-parameters "-option value"
+              --script-parameters '"-option value"'
```

Comment:
Why are these quotes necessary?

Reply:
The first instance looks correct to me, implying the second case of single quoting the double quotes is not necessary.  People should not have to add special formatting to the example to make the linter rule happy.

Reply:
The first instance fails with this error, because without quoting it, the cli itself treats it as an option, not a value....

```
(cli-venv) [05:59 PM] Work-Mac:azure-cli tosin$ az vm show -n "-option-value" -g ova-test
az vm show: error: argument --name/-n: expected one argument
usage: az vm show [-h] [--verbose] [--debug]
                  [--output {json,jsonc,table,tsv,yaml,none}]
                  [--query JMESPATH] [--resource-group RESOURCE_GROUP_NAME]
                  [--name NAME] [--show-details]
                  [--subscription _SUBSCRIPTION] [--ids ID [ID ...]]
(cli-venv) [05:59 PM] Work-Mac:azure-cli tosin$ 
```

Reply:
@tjprescott